src/main/resources/templates/pps/viewPenalties.html

```diff
@@ -16,32 +16,40 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 id="page-title " class="govuk-heading-xl">
-                    <span id="page-title-heading">Late filing penalty for </span>
-                    <span id="company-name" th:text="${companyName}"></span>
+                    <span id="page-title-heading">Penalty summary</span>
                 </h1>
-                <p class="govuk-heading-m" th:text="'Accounts period made up to ' + ${madeUpDate}"></p>
-                <table class="govuk-table">
+
+                <table class="govuk-table" style="width: 800px">
```

Comment:
Is declaring the width here necessary?

Reply:
When testing locally it was too small without declaring the width so I added that in to match sizing in ticket from images